lib/public/views/Runs/RunPerDataPass/RunsPerDataPassOverviewPage.js

```diff
@@ -81,10 +86,22 @@ export const RunsPerDataPassOverviewPage = ({ runs: { perDataPassOverviewModel }
                     NotAsked: () => [commonTitle, tooltip(h('.f3', iconWarning()), 'No data was asked for')],
                 }),
             ),
+
             exportRunsTriggerAndModal(perDataPassOverviewModel, modalModel),
         ]),
         h('.flex-column.w-100', [
-            table(remoteRuns, activeColumns, null, { profile: 'runsPerDataPass' }),
+            table(
+                remoteDetectors.match({
+                    Success: () => remoteRuns,
+                    Other: () => remoteDetectors,
+                }),
```

Comment:
Why is that? Besides, it do not belongs here

Reply:
because previous was wrong, anyway it's revoked